app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt

```diff
@@ -204,4 +207,17 @@ object Utils {
             else -> cardType.NONE
         }
     }
+
+    fun checkLocationSettings(context: Context): Boolean {
```

Comment:
`checkLocationSettings` is a verb, hence, this function should return nothing

If you want this to return boolean, rename to `isLocationEnabled`

Reply:
Updated